elementor/elementor
# Fixes Illegal string offset 'url' in elementor/includes/base/controls-stack.php Issue #13573

Fixes: #13573 

## PR Checklist


## PR Type
What kind of change does this PR introduce?


## Summary

This PR can be summarized in the following changelog entry:

* Fix the `Warning: Illegal string offset 'url' in elementor/includes/base/controls-stack.php on line 1184` warning

## Description
An explanation of what is done in this PR

* Check if the array key exists before setting

## Test instructions
This PR can be tested by following these steps:

* Add a Wrapper URL using dynamic link functionality (e.g. Post URL) with Happy Addons.

See also: 
- https://github.com/dudaster/ele-custom-skin/issues/118
- https://wordpress.org/support/topic/warning-illegal-string-offset-url-after-update/
- https://wordpress.org/support/topic/bug-in-2-9-0-php-warning/
- https://www.google.com/search?q=Warning%3A+Illegal+string+offset+%22%2Fplugins%2Felementor%2Fincludes%2Fbase%2Fcontrols-stack.php%22&oq=Warning%3A+Illegal+string+offset+%22%2Fplugins%2Felementor%2Fincludes%2Fbase%2Fcontrols-stack.php%22

## Quality assurance

- [n/a?] Docs have been added / updated (for bug fixes / features)

Fixes #13573
